App/App.tsx

```diff
@@ -34,21 +32,27 @@ import {
 	TCacheShape,
 } from './stores';
 import { setupAmplitude, track } from './util/amplitude';
-import { IS_SENTRY_SET_UP, RELEASE_CHANNEL } from './util/constants';
 import { sentryError } from './util/sentry';
 
-// Add Sentry if available
-if (IS_SENTRY_SET_UP) {
-	Sentry.init({
-		dsn: Constants.manifest.extra.sentryPublicDsn as string,
-		debug: true,
-	});
+import './localization';
 
-	Sentry.setRelease(RELEASE_CHANNEL);
-	if (Constants.manifest.revisionId) {
-		Sentry.setExtra('sisRevisionId', Constants.manifest.revisionId);
-	}
-}
+// TODO
+// import * as Sentry from 'sentry-expo';
+// import Constants from 'expo-constants';
+// import { IS_SENTRY_SET_UP, RELEASE_CHANNEL } from './util/constants';
```

Comment:
why is this commented out?

Reply:
I coded on Raspberry Pi. And this packages doesn't support ARM so I have to temporary disable it. It should be renable. I must hit my head somewhere. My bad.